blog/forge-v6-release.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Electron Forge v6 Stable Release
+date: 2022-10-26T00:00:00.000Z
+authors:
+  - name: georgexu99
+    url: 'https://github.com/georgexu99'
+    image_url: 'https://github.com/georgexu99.png?size=96'
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: forge-v6-release-blog
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Electron Forge Team is excited to announce that Forge v6 is now available! This stable release contains many significant new features and improvements. Highlights of this release include the addition of our Webpack plugin as well as the introduction of the Forge Core and CLI APIs.
+
+## What Does Switching to Stable Mean?
+
+During our time in a beta release state, our team has been busy refactoring Forge’s code, improving its architecture, creating new features as well as ironing out bugs and issues.
+
+Now that we are done making changes, we’re ready to release a stable version of Forge!
+
+## Upgrading to Forge v6
+
+Forge will try to migrate from Electron Forge v5 automatically as much as possible, but some of it may need to be migrated manually. Importing your settings can be done with the same process as importing an existing Electron app to Forge:
```

Comment:
Forge v5 is pretty dead already compared to Forge v6, since we've been pushing the betas for years now: https://npmtrends.com/@electron-forge/core-vs-electron-forge

The number of people switching from v5 would drastically be outnumbered by:
* Developers using v6 beta and wanting to learn about what "stable" means for the project
* Developers using Builder and wanting to migrate over
* Developers using their own packager/osx-sign/etc. pipeline and looking to migrate over

IMO this blog post should be rewritten to those three crowds rather than for a v5 -> v6 migration.

Reply:
👀 really good point, we haven't addressed points 2 and 3 much.

Reply:
Adding a benefits of using electron forge section- is community engagement and beta testing from larger companies ie. slack something we want to push?

 think forge is ~1k weekly while core/cli are ~30. Should I axe the migration section

Reply:
> Adding a benefits of using electron forge section- is community engagement and beta testing from larger companies ie. slack something we want to push?

I think one of the main points is that Forge will be kept up to date with the newest Electron tooling updates (e.g. universal builds, ASAR integrity, etc.). If you're coming from Builder, you don't have to wait for their maintainers to implement it. If you're coming from your own tooling pipeline, you don't have to figure out how to wire in the features yourselves.

Reply:
> think forge is ~1k weekly while core/cli are ~30. Should I axe the migration section

I think Forge v5 migration isn't that useful. Migrating from an existing project is probably something that you want to include. It should be better documented as well in our Forge docs, since all we have is "run the import script and hope it works".